plugins/cortex/pkg/apis/cortexadmin/cortexadmin.proto

```diff
@@ -37,6 +37,34 @@ service CortexAdmin {
       }
     };
   }
+  rpc GetRule(RuleRequest) returns (QueryResponse) {
+    option (google.api.http) = {
+      get: "/get_rule/{groupName}"
```

Comment:
To follow proper REST style, these should be:
```
GET /rules/{groupName}
```

Reply:
Done!